ai/groq/v0/groq_client_interface_mock_test.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by http://github.com/gojuno/minimock (v3.3.13). DO NOT EDIT.
+// Code generated by http://github.com/gojuno/minimock (v3.3.11). DO NOT EDIT.
```

Comment:
FYI @donch1989 I created a [`make gen-mock`](https://github.com/instill-ai/component/blob/main/Makefile#L8) command to keep the mocks at the latest `minimock` version.